src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -931,74 +927,37 @@ internal string DisplayName
         {
             get
             {
-                string? localizedDisplayName = _sLocalizedDisplayName;
-                if (localizedDisplayName == null && !GlobalizationMode.Invariant)
+                string? localizedDisplayName = NativeName;
+                if (!GlobalizationMode.Invariant && Name.Length > 0)
                 {
-                    if (IsSupplementalCustomCulture)
+                    try
                     {
-                        if (IsNeutralCulture)
+                        const string ZH_CHT = "zh-CHT";
+                        const string ZH_CHS = "zh-CHS";
```

Comment:
Nit: I don't think these consts help with anything... the strings are each used once, and the consts are the exact same letters as in the strings

Reply:
I'll apply your suggestion you mentioned in the comment https://github.com/dotnet/runtime/pull/53468#discussion_r643993864